src/time_travel/patchers/basic_patcher.py

```diff
@@ -4,11 +4,13 @@
 class BasicPatcher(object):
     """Base class for patching time modules."""
     
-    def __init__(self, clock):
+    def __init__(self, clock, events_pool=None):
         """Create the patch."""
         self.clock = clock
-        self.patches = []
+        self.events_pool = events_pool
```

Comment:
The idea is that all patchers can have access to the event_pool and the clock. There is no need to initialize it in all of them. (There will be more classes using the event pool)